python/cuml/cuml/kernel_ridge/kernel_ridge.py

```diff
@@ -247,7 +247,7 @@ def _params_to_cpu(self):
 
     def _attrs_from_cpu(self, model):
         if not isinstance(model.X_fit_, np.ndarray):
-            raise UnsupportedOnGPU
+            raise UnsupportedOnGPU("Sparse inputs are not supported")
```

Comment:
Can we add a code comment with a brief explanation here since the relationship between test and message is not obvious?